dubbo-config/dubbo-config-api/src/test/java/org/apache/dubbo/config/ReferenceConfigTest.java

```diff
@@ -34,7 +34,7 @@ public void testInjvm() throws Exception {
         registry.setAddress("multicast://224.5.6.7:1234");
 
         ProtocolConfig protocol = new ProtocolConfig();
-        protocol.setName("dubbo");
+        protocol.setName("mockprotocol");
```

Comment:
Should we create a test constant for this, as we might be using it many places. It will be easier to  change in case in future if have to do.

Reply:
let's revisit this when we enhance unit test in config-api. In my opinion, config-api's unit test needs to clean up:

1. as you mentioned, not only 'mock protocol' but also other constants are needed to defined as constants.
2. there are some test cases depends on other module which may lead to cycle dependency in the future, for example: 'dubbo-registry-multicast' should not be used by 'dubbo-config-api'.

In this pull request, I solved some of cycle dependencies when I start to use 'dubbo-config-api' in 'dubbo-rpc-dubbo'. We should address others in another delegated change.

Reply:
Agree with you. May be once this PR is merged I could take this cleanup part.